bolt/lib/Rewrite/RewriteInstance.cpp

```diff
@@ -3047,6 +3064,10 @@ void RewriteInstance::selectFunctionsToProcess() {
     return true;
   };
 
+  if (auto *InitBD = BC->getBinaryDataByName("_init"))
```

Comment:
I don't think we need this extra condition

Reply:
Thanks for spotting!